cloud/endagaweb/models.py

```diff
@@ -103,15 +104,20 @@ def new_user_hook(sender, instance, created, **kwargs):
         """
         if created and instance.username != settings.ANONYMOUS_USER_NAME:
             profile = UserProfile.objects.create(user=instance)
-            network = Network.objects.create()
+
+            # To avoid duplicate names while running setup_test_db
```

Comment:
change the comment to 'Add explicit name to avoid duplicate names when running setup_test_db`, to make it clear what you are doing.